Dockerfile.rhtap

```diff
@@ -0,0 +1,16 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 AS builder
+
+WORKDIR /go/src/github.com/stolostron/clusterclaims-controller
+COPY . .
+
+RUN make -f Makefile.prow compile
```

Comment:
This line is returning the following error `GOFLAGS="" go build -o build/_output/manager-clusterclaims ./cmd/clusterclaims/main.go
github.com/gogo/protobuf/proto: /usr/lib/golang/pkg/tool/linux_amd64/compile: signal: killed
unshare: sigprocmask unblock failed: Invalid argument` and not compiling correctly